src/collections/match/operations/match-operation-utils.ts

```diff
@@ -109,12 +109,21 @@ export function verifyMatcherSpec(
     m &&
     Array.isArray(m.senderBranches) &&
     Array.isArray(m.receiverBranches) &&
+    Array.isArray(m.userMatchLocations) &&
     m.senderBranches.every(
       (branchId) => typeof branchId === "string" && branchId.length === 24
     ) &&
     m.receiverBranches.every(
       (branchId) => typeof branchId === "string" && branchId.length === 24
-    )
+    ) &&
+    typeof m.standLocation === "string" &&
+    m.standLocation.length > 0 &&
+    m.userMatchLocations.every(
+      (location) =>
+        typeof location.name === "string" && location.name.length > 0
```

Comment:
Should verify simultaneousMatchLimit also, if it's present
```suggestion
        typeof location.name === "string" &&
        location.name.length > 0 &&
        (location.simultaneousMatchLimit === undefined ||
          (Number.isInteger(location.simultaneousMatchLimit) &&
            location.simultaneousMatchLimit > 0))
```

Reply:
Yup, I concur